### PR TITLE
Setup project to only run x64

### LINKED ISSUE #344: 🚧Setup project to only run x64
### Complete The Item Below

- [X] I have updated the title without removing the 🚧 emoji.

### Description

Set up the project to only run x64.  This will contain various changes for this to occur.

**Reason:**
This is to get a speed benefit.  When running portable/AnyCPU, this will have the application run the application with JIT compilation.  The benefit here is that you do not have to worry about the CPU architecture when running your application.  It just simply works.  The downside is that there will be a performance penalty.  Read more [here](https://stackoverflow.com/questions/53860621/what-are-differences-between-portable-and-win-x64-when-deploying).

**Solution Configs:**
The 3 solution configurations need to be changed from running in **AnyCPU** to **x64**.  This can be done by using the **Visual Studio** or **JetBrains Rider** solution configuration.  This will have to be done with all 3 projects.

The end result should only be the 3 configs below:
- Debug|x64
- Debug-Console|x64
- Release|x64

**CSharp Project Files:**
Any _**Conditions**_ in the csproj files will have to be updated.  This can be done by going to the project settings.  Updating the solution configs might very well do this automatically.

**Workflow Changes:**
The workflows have various setups for the build configuration.  These will have to all be updated.  This also means that the builds will have to be specifically built for `win-x64`.  This can be passed into the reusable workflows in the **Infrastructure** repo.

**Run/Debug Configurations:**
Setup a run/debug configuration that runs the **VelaptorTesting** application.  Currently, one is set up but it is not saved to the **JetBrains Rider** IDE shared settings.  Save this config to the Rider settings to be committed to the repo.
Also, make sure that everything is set up and works for **Visual Studio**

💡The application will have to be tested to make sure that everything runs correctly with the architecture change
💡There will be https://github.com/KinsonDigital/Velaptor/labels/%F0%9F%A7%A8breaking%20changes to anybody that is running x86(32bit) architecture machines. 

### Acceptance Criteria

- [x] Solution configurations updated
  - [x] Velaptor
  - [x] VelaptorTests
  - [x] VelaptorTesting
- [x] CSProj files updated for all projects
- [x] All workflows updated that are required to be updated
  - [x] All build configs updated
  - [x] Specific use of `win-x64` arch when building.
- [x] Run/Debug config for **VelaptorTesting** created and saved for **JetBrains Rider** users
  - [x] Config for Debug-Console
  - [x] Config for Release-Console

### ToDo Items

- [X] Change type labels added to this issue.  Refer to the _**Change Type Labels**_ section below.
- [X] Priority label added to this issue.  Refer to the _**Priority Type Labels**_ section below.
- [X] Issue linked to the correct project _(if applicable)_.
- [X] Issue linked to the correct milestone _(if applicable)_.
- [x] Draft pull request created and linked to this issue _(only required with code changes)_.

### Issue Dependencies

_No response_

### Related Work

_No response_

### Additional Information:

**_<details closed><summary>Change Type Labels</summary>_**

| Change Type         | Label                |
|---------------------|----------------------|
| Bug Fixes           | `🐛bug` |
| Breaking Changes    | `🧨breaking changes` |
| New Feature         | `✨new feature` |
| Workflow Changes    | `workflow` |
| Code Doc Changes    | `🗒️documentation/code` |
| Product Doc Changes | `📝documentation/product` |
</details>

**_<details closed><summary>Priority Type Labels</summary>_**

| Priority Type       | Label             |
|---------------------|-------------------|
| Low Priority        | `low priority`    |
| Medium Priority     | `medium priority` |
| High Priority       | `high priority`   |
</details>

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

### PR DESCRIPTION
**Pull request by CalvinWilkinson**
<h1 style="font-weight:bold" align="center">Preview Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>

- [x] The **_head(source)_** branch for this pull request is a **_preview feature_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/feature/<issue-num>-<description>
      Example: preview/feature/123-my-preview-feature
      ```
    </details>

- [x] The **_head(source)_** branch for this pull request is created from a **_preview release_** branch, with the correct naming syntax, in the following example:
  <details closed><summary>Example</summary>

    ``` xml
    Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
    Example: preview/v1.2.3-preview.4
    ```
  </details>

- [x] The **_base(target)_** branch for this pull request is a **_preview release_** branch, with the correct naming syntax, in the following example:
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.

    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
      Example: preview/v1.2.3-preview.4
      ```
    </details>

💡For more information on branching, refer to the project's [branching documentation](../../Documentation/Branching.md).

- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] Pull request title matches the title of the associated issue.

- [x] A `preview` label has been added to the pull request.

- [x] The labels attached to this PR match the labels attached to the associated issue.

- [x] Project assigned matches the project of the associated issue.

- [x] Milestone assigned matches the milestone of the associated issue.

- [x] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out and let's discuss it.

- [x] I have manually tested my code changes to the best of my ability.

- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - Status checks run for this pull request when merging into the **_preview/v\*.\*.\*-preview.\*_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

File: .github/workflows/preview-release.yml [hunk 1]
```diff
@@ -133,12 +133,14 @@ jobs:
       run: dotnet build "${{ github.workspace }}/${{ env.PROJECT_NAME }}/${{ env.PROJECT_NAME }}.csproj" `
         -c ${{ env.BUILD_CONFIG }} `
         --no-restore
```
Comment (KinsonDigitalAdmin) [inline .github/workflows/preview-release.yml:135]: Add a back tick to the end of this line for multiline PowerShell

```suggestion
        --no-restore `
```

File: .github/workflows/preview-release.yml [hunk 2]
```diff
@@ -133,12 +133,14 @@ jobs:
...
+        -a win-x64
 
     - name: Execute Unit Tests
       run: dotnet test "${{ github.workspace }}/Testing/${{ env.PROJECT_NAME }}Tests/${{ env.PROJECT_NAME }}Tests.csproj" `
         -c ${{ env.BUILD_CONFIG }} `
         --collect:"XPlat Code Coverage" `
         --no-restore
```
Comment (KinsonDigitalAdmin) [inline .github/workflows/preview-release.yml:142]: Add a back tick to the end of this line for multiline PowerShell

```suggestion
        --no-restore `
```

**Review by KinsonDigitalAdmin — CHANGES_REQUESTED** [review]

File: .run/Publish VelaptorTesting (Release Linux).run.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Publish VelaptorTesting (Debug Linux)" type="DotNetFolderPublish" factoryName="Publish to folder">
```
Comment (KinsonDigitalAdmin) [inline .run/Publish VelaptorTesting (Release Linux).run.xml:2]: Change this to say Release instead of Debug

```suggestion
  <configuration default="false" name="Publish VelaptorTesting (Release Linux)" type="DotNetFolderPublish" factoryName="Publish to folder">
```

**Review by KinsonDigitalAdmin — CHANGES_REQUESTED** [review]

**Review by KinsonDigitalAdmin — APPROVED** [review]
